upload/templates/modeladmin/upload/package/inspect.html

```diff
@@ -43,7 +43,11 @@ <h3>{% trans "Reports" %}</h3>
                     {% if validation_errors.xml_format_error %}
                         <span class='help-block help-critical'>{% trans "XML file has errors." %}</a></span>
                     {% else %}
-                        <span class='help-block help-info'>{% trans 'There are no errors.' %}</span>
+                        {% if validation_errors.package_file_error %}
+                            <span class='help-block help-info'>{% trans 'This valiation was not performed.' %}</span>
```

Comment:
A rigor significa que não foi e nem vai ser. 
Sugestão: `This validation has not been performed yet`
Corrigir a grafia de `valiation`